torch/utils/data/dataframes_pipes.ipynb

```diff
@@ -0,0 +1,437 @@
+      "1  1  1   98.3\n",
+      "2  2  2  199.3\n",
+      "   i  j      y\n",
+      "0  3  0  297.3\n",
+      "1  4  1  398.3\n",
+      "2  5  2  499.3\n",
+      "   i  j\n",
+      "0  0  0\n",
+      "   i  j\n",
+      "1  1  1\n",
+      "   i  j\n",
+      "2  2  2\n",
+      "   i  j\n",
+      "3  3  0\n",
+      "   i  j\n",
+      "0  4  1\n",
+      "   i  j\n",
+      "1  5  2\n",
+      "   i  j\n",
+      "2  6  0\n",
+      "   i  j\n",
+      "3  7  1\n"
+     ]
+    }
+   ],
+   "source": [
+    "dp0 = get_dataframes_pipe(range = 8, dataframe_size = 4)\n",
+    "dp = get_dataframes_pipe(range = 6, dataframe_size = 3)\n",
+    "dp['y'] = dp.i * 100 + dp.j - 2.7\n",
+    "dp = dp.concat(dp0)\n",
+    "for i,v in enumerate(dp.raw_iterator()):\n",
+    "    print(v)"
+   ]
+  },
+  {
+   "source": [
+    "`unbatch` of `list` with DataFrame works similarly to regular unbatch.\n",
+    "Note: DataFrame sizes might change"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "   i  j      y      z\n0  0  0   -2.7 -102.7\n1  1  1   98.3   -1.7\n2  2  2  199.3   99.3\n0  3  0  297.3  197.3\n1  4  1  398.3  298.3\n   i  j      y      z\n2  5  2  499.3  399.3\n0  6  0  597.3  497.3\n1  7  1  698.3  598.3\n2  8  2  799.3  699.3\n0  9  0  897.3  797.3\n    i  j       y       z\n1  10  1   998.3   898.3\n2  11  2  1099.3   999.3\n0  12  0  1197.3  1097.3\n1  13  1  1298.3  1198.3\n2  14  2  1399.3  1299.3\n    i  j       y       z\n0  15  0  1497.3  1397.3\n1  16  1  1598.3  1498.3\n2  17  2  1699.3  1599.3\n"
+     ]
+    }
+   ],
+   "source": [
+    "dp = get_dataframes_pipe(range = 18, dataframe_size = 3)\n",
+    "dp['y'] = dp.i * 100 + dp.j - 2.7\n",
+    "dp = dp.batch(5).batch(3).batch(1).unbatch(unbatch_level = 3)\n",
+    "\n",
+    "# Here is bug with unbatching which doesn't detect DF type.\n",
+    "dp['z'] = dp.y - 100\n",
+    "\n",
+    "for i in dp.raw_iterator():\n",
+    "    print(i)"
+   ]
+  },
+  {
+   "source": [
+    "`map` applied to individual rows, `nesting_level` argument used to penetrate batching"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "[       i      j\n0  1.111  1.111\n1  1.112  1.112\n2  1.113  1.113\n0  1.114  1.111\n1  1.115  1.112]\n[       i      j\n2  1.116  1.113\n0  1.117  1.111\n1  1.118  1.112\n2  1.119  1.113\n0  1.120  1.111]\n[(1.111, 0), (1.112, 1), (1.113, 2), (1.114, 0), (1.115, 1)]\n[(1.116, 2), (1.117, 0), (1.118, 1), (1.119, 2), (1.12, 0)]\n"
+     ]
+    }
+   ],
+   "source": [
+    "dp = get_dataframes_pipe(range = 10, dataframe_size = 3)\n",
+    "dp = dp.map(lambda x: x + 1111)\n",
```

Comment:
I probably miss something, but how is `map` over dataframe implemented? only found `dataframes_filter` but not for `map`.